### ISSUE TITLE
untracked files in go-ontology repo

### ISSUE THREAD

**Issue by krchristie** [geneontology/go-ontology#14098]
At the Protege workshop, Chris said he would add the files listed in the results of a git status command below to the list of files to ignore. The list should perhaps include everything in the reports/ directory, not just the two files in that directory listed here.

```
~/repos/go-ontology(master) $ git status
On branch master
Your branch is up-to-date with 'origin/master'.
Untracked files:
  (use "git add <file>..." to include in what will be committed)

src/ontology/disjoint_in_taxon.owl
src/ontology/enhanced.owl
src/ontology/reports/equivalent-classes-violation.csv
src/ontology/reports/trailing-whitespace-violation.csv

nothing added to commit but untracked files present (use "git add" to track)
```

**Comment by krchristie** [comment]
list with additional untracked files after running 'make test' command line:

```
Untracked files:
  (use "git add <file>..." to include in what will be committed)

	src/ontology/change-report.txt
	src/ontology/disjoint_in_taxon.owl
	src/ontology/enhanced.owl
	src/ontology/go-edit.obo-check
	src/ontology/go-lastrelease.owl
	src/ontology/reports/equivalent-classes-violation.csv
	src/ontology/reports/trailing-whitespace-violation.csv
```

**Comment by cmungall** [comment]
Closed via #14102

**Comment by krchristie** [comment]
Thanks Chris!
